cmd/machine-api-operator/start.go

```diff
@@ -9,6 +9,7 @@ import (
 	"strconv"
 
 	osconfigv1 "github.com/openshift/api/config/v1"
+
```

Comment:
This new line seems to be randomly added, can we remove it please?

Reply:
Sure thing, updated